akka-http-tests/src/test/java/akka/http/javadsl/marshalling/sse/EventStreamMarshallingTest.java

```diff
@@ -44,7 +45,7 @@ public void testToEventStream() {
         final ByteString expectedEntity = events
                 .stream()
                 .map(e -> ((akka.http.scaladsl.model.sse.ServerSentEvent) e).encode())
-                .reduce(ByteString.empty(), ByteString::concat);
+                .reduce(ByteString$.MODULE$.empty(), ByteString::concat);
```

Comment:
(can be improved once Akka is released with https://github.com/akka/akka/pull/26931)

Reply:
Yep, filed https://github.com/akka/akka-http/issues/2530 so we don't forget.